src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -11,7 +11,7 @@
 from mgr_module import MgrModule
 
 from .controllers.auth import Auth
-from .tools import load_controllers
+from .tools import load_controllers, _json_error_page
```

Comment:
please remove the `_` prefix from the function name since it's being used externally to the tools module